### ISSUE TITLE
wechat-uos: "/no" in messages is displayed as spaces

### ISSUE THREAD

**Issue by luochen1990** [NixOS/nixpkgs#332400]
### Describe the bug

"/no" in messages is displayed as spaces

### Steps To Reproduce
1. send "abc/nodef" to yourself
2. the message is displayed as "abc   def"
3. check the message from mobile, it is "abc/nodef" which is correct. so we confirm that this is a display issue

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
- system: `"x86_64-linux"`
- host os: `Linux 6.9.7-zen1, NixOS, 24.11 (Vicuna), 24.11.20240706.5041afc`
- multi-user?: `yes`
- sandbox: `yes`
- version: `nix-env (Nix) 2.23.1`
- nixpkgs: `/nix/store/xh78jcd1jy5kx3pvjf4jdyvwl731w47z-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by Cryolitia** [comment]
This is unlikely to be a nixpkgs issue, and you may need to consider whether it can be reproduced on other distributions and report it upstream.

**Comment by luochen1990** [comment]
This issue is solved at version 4.0.0.23
